src/django_rich/runner.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import sys
+import unittest
+from unittest.result import STDERR_LINE, STDOUT_LINE
```

Comment:
Mypy is saying:

```
src/django_rich/runner.py:5: error: Module "unittest.result" has no attribute "STDERR_LINE"  [attr-defined]
src/django_rich/runner.py:5: error: Module "unittest.result" has no attribute "STDOUT_LINE"  [attr-defined]
```

I can see these exist though - probably the type stubs just don't include those variables. Ignore the errors with:

```suggestion
from unittest.result import STDERR_LINE, STDOUT_LINE  # type: ignore [attr-defined]
```